index.Rmd

```diff
@@ -134,8 +139,11 @@ system("cp -r previous_versions/* docs/previous_versions/")
 file.copy("images/logos/wide_format.png", "docs/wide_format.png", overwrite = TRUE)
 
 # Copy pdf file if needed
-if(file.exists("ismaykim.pdf"))
-  file.copy("ismaykim.pdf", "docs/ismaykim.pdf", overwrite = TRUE)
+if(!file.exists("moderndive.pdf")){
+  file.copy(from = "docs/moderndive.pdf", 
```

Comment:
Any reason why we set `output_dir: "docs"` in `_bookdown.yml` and then do this `file.copy()` here? Couldn't we just `output_dir: "ROOT"`?